ballerina-tests/tests/47_unary_timestamp_service.bal

```diff
@@ -27,34 +27,34 @@ service "TimestampService" on ep47 {
             name: value,
             time: check time:utcFromString("2007-12-03T10:15:30.120Z")
         };
-        check caller->sendGreeting(greeting);
-        check caller->complete();
+        checkpanic caller->sendGreeting(greeting);
+        checkpanic caller->complete();
     }
 
-    remote function exchangeGreeting(TimestampServiceGreetingCaller caller, Greeting value) returns error? {
+    remote function exchangeGreeting(TimestampServiceGreetingCaller caller, Greeting value) {
         Greeting greeting = {
             name: value.name,
-            time: check time:utcFromString("2008-12-03T11:15:30.120Z")
+            time: checkpanic time:utcFromString("2008-12-03T11:15:30.120Z")
```

Comment:
```suggestion
            time: check time:utcFromString("2008-12-03T11:15:30.120Z")
```

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/606/commits/c32ef7e98e2bfaa4df966a96025d16d755dfb1cd